agent/ansible/pbench/agent/galaxy.yml

```diff
@@ -0,0 +1,57 @@
+### REQUIRED
+
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: pbench
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: agent
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.2
```

Comment:
Seems like the Makefile should derive the version number it uses from this file?